.github/workflows/unittesting.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI Unit tests
+
+on:
+  push:
+    branches: [ dev, master, main ]
```

Comment:
Could you remind me what's the `main` branch does? Are we migrating master to main?

Reply:
We would be migrating soon from master to main and deprecate the master branch - ref: https://github.com/github/renaming